examples/bert/bert_cuda_gpu.json

```diff
@@ -76,7 +76,6 @@
             }
         },
         "evaluator": "common_evaluator",
```

Comment:
What will happend if we remove EP but no any system configs are given like this example? Seems it will use CPU and CPU EP, right?

Also similar questions in:
1. What if only `device` is given in accelerators. -> get_available_providers? 
2. What if only `execution_provider` is given in accel. -> infer device but ignore CPU as you did?
3. What if no `accelerators` config is given like in this example.

Could you help double check? Also update the related docs if needed.